src/styles/blocks/page.scss

```
@@ -3,5 +3,6 @@
 
   &__body {
     margin: 0;
+    --duration: 300ms;
```

```suggestion
    --duration: 300ms;
```
In my opinion in this case scss variables is more suitable